Release notes update (#4374)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/ReleaseNotesKogito.1.29.0.Final/ReleaseNotesKogito.1.29.0.Final.adoc

https://kiegroup.github.io/kogito-docs/serverlessworkflow/1.29.0.Final/release_notes.html[New features on 1.29].

== {PRODUCT} supporting services

=== Jobs Service

> Starting from this release, Jobs Service allows that multiple instances of the service can be running at the same time.

Reviewer: ```suggestion
Starting from this release, jobs service allows multiple instances of the service to run simultaneously.
```

> This is done via primary and stand-by election, where only one would be active at a given time.

Reviewer: ```suggestion
This is done via primary and stand-by elections, where only one would be active at a given time.
```
